packages/itwinui-css/src/button/active.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-button-active {
```

Comment:
This mixin seems to also be there in `base.scss`. One of them is a duplicate and can be removed?

Reply:
You are correct.  After our call earlier today where we discussed `base.scss` being set to default button variant values, I made sure the `@mixin iui-button-active` within `base.scss` is set to default values, and then moved the borderless active mixin to `borderless.scss`.